pages/mesosphere/dcos/services/cassandra/2.7.0-3.11.4/operations/index.md

```diff
@@ -81,7 +81,7 @@ dcos {{ model.packageName }} --name=<service-name> pod replace node-0
 The output of a recovery plan is as follows:
 
 ```bash
-$ dcos {{ model.packageName }} --name=<service-name> plan show recovery
+dcos {{ model.packageName }} --name=<service-name> plan show recovery
```

Comment:
The `$` signifies that this is a command that needs to be run so we'd need to add this back

Reply:
Done